### DISCUSSION TITLE
White Label

### DISCUSSION THREAD

**Discussion by etmartinkazoo** [mnfst/llm-gateway#222] [Feature request]
First, thank you for this project. It makes a TON of sense for most people's use cases.

Would it be possible to add a field to white label the logo and favicon?

**Comment by brunobuddy** [comment]
@etmartinkazoo I will check if that is something that can be done easily

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
